src/makeInfinityAdder.js

```diff
@@ -6,7 +6,20 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  return function adder(num) {
+    if (num === undefined) {
+      const result = sum;
```

Comment:
Why are you copying the value here?

Reply:
@alexander-ignatow I need to clear value for empty argument call, so I keep this value in new variables, then clear sum and return result;